onnxruntime/test/optimizer/graph_transform_test_builder.h

```diff
@@ -48,6 +48,9 @@ struct IsTypeQuantLinearCompatible<int16_t> : std::true_type {};
 template <>
 struct IsTypeQuantLinearCompatible<uint16_t> : std::true_type {};
 
+template <>
+struct IsTypeQuantLinearCompatible<int32_t> : std::true_type {};
```

Comment:
Is this strictly required? The ONNX spec says that the QuantizeLinear operator does not support the int32 type. Only Dequantize linear supports int32.

Reply:
good point, I should change it another way, in the AddQDQNodePair method.